Doc/tutorial/controlflow.rst

```diff
@@ -163,14 +163,14 @@ arguments.  In chapter :ref:`tut-structures`, we will discuss in more detail abo
 :keyword:`!break` and :keyword:`!continue` Statements, and :keyword:`!else` Clauses on Loops
 ============================================================================================
 
-The :keyword:`break` statement, like in C, breaks out of the innermost enclosing
+The :keyword:`break` statement breaks out of the innermost enclosing
 :keyword:`for` or :keyword:`while` loop.
 
-Loop statements may have an :keyword:`!else` clause; it is executed when the loop
-terminates through exhaustion of the iterable (with :keyword:`for`) or when the
-condition becomes false (with :keyword:`while`), but not when the loop is
-terminated by a :keyword:`break` statement.  This is exemplified by the
-following loop, which searches for prime numbers::
+The :keyword:`for` loop statements may have an extra :keyword:`!else` clause;
+it is executed when the loop terminates through exhaustion of the iterable
+(with :keyword:`for`) or when the condition becomes false (with :keyword:`while`),
+but not when the loop is terminated by a :keyword:`break` statement.
```

Comment:
I find the original language here rather hard to parse because the concepts are quite unfamiliar (a loop that terminates through exhaustion of an iterable...).

Perhaps:

A for or while loop can include an else clause. 

In a for loop, the else clause is executed after the loop reaches its final iteration. In a while loop, it's executed after the loop's condition becomes false. 

In either kind of loop, the else clause is **not** executed if the loop was terminated by a break.